pkg/cloudprovider/providers/azure/azure_backoff.go

```diff
@@ -26,11 +26,25 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
+// getorCreateRequestBackoff returns a new Backoff object steps = 1
+// This is to make sure that the requested command executes
+// at least once
+func (az *Cloud) getorCreateRequestBackoff() (resourceRequestBackoff wait.Backoff) {
+	if az.CloudProviderBackoff {
+		return az.resourceRequestBackoff
+	}
+	resourceRequestBackoff = wait.Backoff{
+		Steps: 1,
+	}
+
+	return resourceRequestBackoff
+}
+
```

Comment:
To my understanding the back off object is always there, maybe with zeros as values but it will be there. The first check will always result in true..

Reply:
The  az.CloudProviderBackoff is actually a bool that is a config and not a real Backoff object, it is set based on config provided through Azure.json

Reply:
The az.CloudProviderBackoff is actually a bool that is a config and not a real Backoff object, it is set based on config provided through Azure.json

Reply:
`az.CloudProviderBackoff ` is a boolean object. 

If it is true, the az.resourceRequestBackoff will always be >= 1, 
We make it to 1 when az.CloudProviderBackoff is false. So we will execute at least once.

The first check could be false.